File: Java/IntegerListFromUsp/src/Exercicio01.java
import java.util.Scanner;

public class Exercicio01 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int numero = -1;
        while (numero != 0) {
            scanner.reset();
            System.out.print("Digite um número: ");
            numero = scanner.nextInt();
            int quadrado = numero * numero;
            System.out.println("O quadrado de %d é %d".formatted(numero, quadrado));
        }
    }
}

File: CPP/IntegerListFromUsp/src/Exercicio01.cpp
#include <iostream>

using namespace std;

int main() {
    int numero = -1;
    while (numero != 0) {
        cout << "Digite um número (0 para sair): ";
        cin >> numero;
        if (numero != 0) {
            cout << "O quadrado de " << numero << " é " << numero * numero << endl;
        }
    }
    return 0;
}
